Tests/MapboxMapsTests/Annotations/AnnotationOrchestratorImplTests.swift

```diff
@@ -181,4 +179,195 @@ final class AnnotationOrchestratorImplTests: XCTestCase {
+        let annotationManagerLayerId = "managerId"
+        let annotationManager = MockAnnotationManager()
+        annotationManager.$layerId.getStub.defaultReturnValue = annotationManagerLayerId
+        factory.makePointAnnotationManagerStub.defaultReturnValue = annotationManager
+        longPressGestureRecognizer.getStateStub.defaultReturnValue = .began
+        _ = impl?.makePointAnnotationManager(id: annotationManagerLayerId, layerPosition: nil, clusterOptions: nil)
+
+        // when
+        longPressGestureRecognizer.sendActions()
+
+        // then
+        XCTAssertEqual(mapFeatureQueryable.queryRenderedFeaturesAtStub.invocations.count, 1)
+        XCTAssertEqual(mapFeatureQueryable.queryRenderedFeaturesAtStub.invocations.first?.parameters.options?.layerIds, [annotationManagerLayerId])
+    }
+
+    func testAnnotationOrchestratorDragGestureBeginNotifiesAnnotationManagersAboutQRFSuccessForAllManagersExceptPoint() {
+        // given
+        let annotationManagerLayerId = "managerId"
+        let annotationManager = MockAnnotationManager()
+        factory.makePolygonAnnotationManagerStub.defaultReturnValue = annotationManager
+        factory.makeCircleAnnotationManagerStub.defaultReturnValue = annotationManager
+        factory.makePolylineAnnotationManagerStub.defaultReturnValue = annotationManager
+        longPressGestureRecognizer.getStateStub.defaultReturnValue = .began
+        _ = impl?.makePolygonAnnotationManager(id: annotationManagerLayerId, layerPosition: nil)
+
+        // when
+        longPressGestureRecognizer.sendActions()
+        let qrfCompletions: (Result<[QueriedFeature], Error>) -> Void = (mapFeatureQueryable.queryRenderedFeaturesAtStub.invocations.first?.parameters.completion)!
+        let featureIdentifier = "feature"
+        let feature = QueriedFeature.init(
+            __feature: MapboxCommon.Feature.init(
+                identifier: featureIdentifier as NSObject,
+                geometry: MapboxCommon.Geometry(Point(.init(latitude: 0, longitude: 0))),
+                properties: [:]),
+            source: "feature-source",
+            sourceLayer: nil,
+            state: "feature-state"
+        )
+
+        qrfCompletions(.success([feature]))
+
+        // then
+        XCTAssertEqual(annotationManager.handleDragBeginStub.invocations.count, 1)
+        XCTAssertEqual(annotationManager.handleDragBeginStub.invocations.first!.parameters.first, featureIdentifier)
+    }
+
+    func testAnnotationOrchestratorDragGestureBeginNotifiesAnnotationManagersAboutQRFSuccessForPointManager() {
+        // given
+        let annotationManagerLayerId = "managerId"
+        let annotationManager = MockAnnotationManager()
+
+        factory.makePointAnnotationManagerStub.defaultReturnValue = annotationManager
+        longPressGestureRecognizer.getStateStub.defaultReturnValue = .began
+        _ = impl?.makePointAnnotationManager(id: annotationManagerLayerId, layerPosition: nil, clusterOptions: nil)
```

Comment:
Maybe make an extension method `makePointAnnotationManager(id:layerPosition)` so you can put all of them into a single test